lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -90,7 +90,7 @@ public function register_routes() {
 						'force' => array(
 							'type'        => 'boolean',
 							'default'     => false,
-							'description' => __( 'Whether to bypass Trash and force deletion.', 'default' ),
+							'description' => __( 'Whether to bypass Trash and force deletion.', 'gutenberg' ),
 						),
```

Comment:
All of the strings with changed textdomain in this file come from Core, so the use of `default` is intentional.